README.md.template

```diff
@@ -28,4 +28,4 @@ Sometimes, I also blog.
 ---
 
 If you like what I do, you can always 
-♥ [sponsor](https://github.com/sponsors/patriksvensson) me.
\ No newline at end of file
+♥ [sponsor](https://github.com/sponsors/patriksvensson) me.
```

Comment:
Ugh. Just noticed that the GitHub online editor added this newline. I can remove.